sdk/eventhub/azure-eventhub/azure/eventhub/_producer_client.py

```diff
@@ -201,7 +201,9 @@ def __init__(
             network_tracing=kwargs.get("logging_enable"),
             **kwargs
         )
-        self._auth_uri = f"sb://{self._address.hostname}{self._address.path}"
+        # remove port from auth_uri (if present)
+        auth_uri_hostname = self._address.hostname.split(":")[0]
+        self._auth_uri = f"sb://{auth_uri_hostname}{self._address.path}"
```

Comment:
not super related to this pr, but I think this can be removed here b/c it's in client base? I know I added it to consumer client separately b/c it's a different uri, but I don't remember why I added it to producer client as well.